packages/server-web/src/client/component/TodoItem.vue

```diff
@@ -91,11 +92,13 @@ export default {
       return this.$props.todo.title;
     },
     todoLabels() {
-      return this.$props.todo.labels || [];
+      return this.$props.todo.labels.map(stringifyLabel) || [];
     },
     titleWithLabels: function() {
       const { todo } = this.$props;
-      return `${todo.title} ${todo.labels.join(" ")}`;
+      console.log({ labelsArgs: todo.labels.map(l => l.args) });
```

Comment:
debug output?

Reply:
> Now even works on Firefox!! Great!
> Except for two minor problems: the checkmark is missing on firefox and the script editor doesn't have a background:
> ![image](https://user-images.githubusercontent.com/143687/47446106-e0b4ae00-d790-11e8-803e-559b2b082718.png)
> 
> I wouldn't require fixing those two, as it's already a huge improvement to the current "not working at all" state. But at least the logo would be nice to have.

I've added a border around the script-editor as I was seeing a difference in the background on my computer. Probably related to monitor?

The checkmark in the logo seems to be an issue with SVG filters. I've played around with it a bit, but could not get it to work / look like the logo I've set up in Sketch. I got rid of the shadow for now, making it look "okay" in both browsers.